### ISSUE TITLE
transform_column, using .apply(), operates elementwise in a column individually

### ISSUE THREAD

**Issue by zbarry** [pyjanitor-devs/pyjanitor#84]
![image](https://user-images.githubusercontent.com/4762711/49702510-d6c2ff00-fbc6-11e8-9840-d964b000cb39.png)

This precludes doing something nice like normalizing a column into probabilities as in:

`df['col'] = df['col'] / df['col'].sum()`

**Comment by ericmjl** [comment]
I might be sounding crazy here, but what about a `janitor.statistics` submodule? 👀

**Comment by szuckerman** [comment]
I think this is an interesting concept, but I'm struggling to think of practical examples to be able to think about implementing this.

R's `dplyr` package has a [`rowwise` function](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/rowwise) that basically does what you're discussing (even though I don't think the 'Details' section of that link makes any sense).

**Comment by zbarry** [comment]
```R

# NOT RUN {
df <- expand.grid(x = 1:3, y = 3:1)
df_done <- df %>% rowwise() %>% do(i = seq(.$x, .$y))
df_done
df_done %>% summarise(n = length(i))
# }
```

R is such a beautiful language. 

I had a practical example months ago that spawned this idea from what I was directly working on, though that's well phased out of my brain at this point......

**Comment by jcvall** [comment]
I like to add a normalize and standardize function to pyjanitor.  I guess we can do this with transform, but I like the readability of the code over a lambda function to make it work.  Like normalize_columns() or standardize_columns(), since we already have expand_columns() (aka. dummy codes) and max_min_scale().  This will put preprocessing into the hands of a more beginners, even though I have noting against sklearn.  I will say, doing a .fit() call on preprocessing bothers me, and I dont know why?  What are your thoughts?

**Comment by fireddd** [comment]
Hi, I would like to see if I can contribute to this issue, can you give a better idea of what's expected here?

**Comment by ericmjl** [comment]
@fireddd I think we've lost context on the original issue, so I'm going to close it. Thanks for picking up on the thread though!
